### PR TITLE
Refactor person to customer

### LINKED ISSUE #227: Refactor person to customer

### PR DESCRIPTION
**Pull request by awdse22**
Close #227

### CODE REVIEW

File: src/test/java/seedu/sellsavvy/logic/parser/MarkOrderCommandParserTest.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 /**
  * As we are only doing white-box testing, our test cases do not cover path variations
  * outside of the MarkOrderCommand code. For example, inputs "1" and "1 abc" take the
- * same path through the DeletePersonCommand, and therefore we test only one of them.
+ * same path through the DeleteCustomerCommand, and therefore we test only one of them.
```
Comment (FooChao) [inline src/test/java/seedu/sellsavvy/logic/parser/MarkOrderCommandParserTest.java:15]: is this an error?

File: src/test/java/seedu/sellsavvy/logic/parser/UnmarkOrderCommandParserTest.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 /**
  * As we are only doing white-box testing, our test cases do not cover path variations
  * outside of the UnmarkOrderCommand code. For example, inputs "1" and "1 abc" take the
- * same path through the DeletePersonCommand, and therefore we test only one of them.
+ * same path through the DeleteCustomerCommand, and therefore we test only one of them.
```
Comment (FooChao) [inline src/test/java/seedu/sellsavvy/logic/parser/UnmarkOrderCommandParserTest.java:15]: same as above.

**Review by FooChao — APPROVED** [review]

File: src/test/java/seedu/sellsavvy/logic/parser/MarkOrderCommandParserTest.java (hunk 1 above)
Comment (awdse22) [reply src/test/java/seedu/sellsavvy/logic/parser/MarkOrderCommandParserTest.java:15]: changed

File: src/test/java/seedu/sellsavvy/logic/parser/UnmarkOrderCommandParserTest.java (hunk 2 above)
Comment (awdse22) [reply src/test/java/seedu/sellsavvy/logic/parser/UnmarkOrderCommandParserTest.java:15]: changed

**Review by kaixuan477 — APPROVED** [review]
Looks good.

**Pull request merged**